jobs/e2e_node/image-config-serial-cpu-manager.yaml

```diff
@@ -8,7 +8,7 @@ images:
     # Using `n1-standard-4` to enable CPU manager node e2e tests.
     machine: n1-standard-4
   cos-stable1:
-    image_regex: cos-stable-73-11647-510-0 # docker v18.09.7, deprecated after 2020-06-19
+    image_regex: cos-73-11647-534-0 # docker v18.09.7, deprecated after 2020-06-19
```

Comment:
Is the comment still applicable for this image?

Reply:
> I'm wondering, should we update these cos images to `cos-stable-81-12871-103-0` ?

We probably should, but not right now. I'd prefer to fix test failures first. 
 
> Perhaps you could elaborate on why you chose `cos-73-11647-534-0`?

I've chosen it as it belongs to the same milestone(73) as cos-stable-73-11647-510-0. this means the changes between images are minimal and the replacement would probably not introduce new issues.